### PR TITLE
Add new errors

### PR DESCRIPTION
**Pull request by Giovannini**

### CODE REVIEW

File: src/project/src/test/scala/io/giovannini/EvenFibonnacciTest2.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ class EvenFibonnacciTest2 extends WordSpec {
     }
 
     "return the right value for n = 2" in {
-      assert(EvenFibonnacci.fibonnacci(2) == Seq(1, 1))
+      assert(EvenFibonnacci.fibonnacci(2) == Seq(2))
```
Comment (khand19) [inline src/project/src/test/scala/io/giovannini/EvenFibonnacciTest2.scala:14]: 🙅 List(1, 1) did not equal List(2)

File: src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala [hunk 2]
```diff
@@ -7,7 +7,7 @@ class EvenFibonnacciTest extends WordSpec {
   "fibonnacci" should {
 
     "return the right value for n = 1" in {
-      assert(EvenFibonnacci.fibonnacci(1) == Seq(1))
+      assert(EvenFibonnacci.fibonnacci(1) == Seq())
```
Comment (khand19) [inline src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala:10]: 🙅 List(1) did not equal List()

File: src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala [hunk 3]
```diff
@@ -16,7 +16,7 @@ class EvenFibonnacciTest extends WordSpec {
 
     "return the right value for n = 10" in {
       assert(
```
Comment (khand19) [inline src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala:18]: 🙅 List(1, 1, 2, 3, 5, 8, 13, 21, 34, 55) did not equal List(1, 1, 2, 3, 4, 8, 13, 21, 34, 55)

**Comment by Giovannini** [comment]
test this please

File: src/project/src/test/scala/io/giovannini/EvenFibonnacciTest2.scala (hunk 1 above)
Comment (khand19) [inline src/project/src/test/scala/io/giovannini/EvenFibonnacciTest2.scala:14]: 🙅 List(1, 1) did not equal List(2)

File: src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala (hunk 2 above)
Comment (khand19) [inline src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala:10]: 🙅 List(1) did not equal List()

File: src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala (hunk 3 above)
Comment (khand19) [inline src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala:18]: 🙅 List(1, 1, 2, 3, 5, 8, 13, 21, 34, 55) did not equal List(1, 1, 2, 3, 4, 8, 13, 21, 34, 55)

**Pull request closed without merging**